packages/manager/modules/pci/src/projects/project/notebooks/notebooks.html

```diff
@@ -92,6 +92,16 @@
         >
             <span data-ng-bind=":: $row.durationString"></span>
         </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_notebook_list_creator' | translate"
+            data-property="user"
+            data-type="string"
+            data-searchable
+            data-filterable
+            data-sortable
+        >
+            <span data-ng-bind="::$value"></span>
```

Comment:
**suggestion:** Not needed

```suggestion
```

Reply:
**thought:** Following table now reach 10 columns. Do you know if its planned to improve the way that all data are displayed to the user?

Reply:
Not to my knowledge, no